llvm/test/CodeGen/AMDGPU/callbr.ll

```diff
@@ -0,0 +1,70 @@
+; RUN: rm -rf %t && split-file %s %t
```

Comment:
Try to avoid split-file. It's kind of useful in cases with poor error handling (as in they exit early without reporting all errors in the file). Maintaining codegen tests using it is unnecessarily difficult.

Reply:
Is there another way to be able to compare against other code? Because I thought it's useful to make sure that the generated callbr code is equivalent to just using call. But the "just call code" shouldn't be tested itself at this point and should only serve as a reference. Is there a way to put this somewhere else than in the current file?

Reply:
ok, adapted